src/Raven.Server/Config/Categories/EmbeddedConfiguration.cs

```diff
@@ -0,0 +1,47 @@
+using System;
+using System.ComponentModel;
+using Raven.Server.Config.Attributes;
+
+namespace Raven.Server.Config.Categories
+{
+    public class EmbeddedConfiguration : ConfigurationCategory
+    {
+        private int? _legacyParentProcessId;
+        private int? _parentProcessId;
+
+        [Description("Watch the parent process id and exit when it exited as well (for tests)")]
+        [DefaultValue(null)]
+        [ConfigurationEntry("Testing.ParentProcessId", ConfigurationEntryScope.ServerWideOnly)]
```

Comment:
I think that we should add support for multiple keys in configuration entry. We do not really care from which key it was set probably or drop 'Testing.ParentProcessId' entirely.

Reply:
Good idea

Reply:
No, we want a _single_ property, not LegacyParentProcessId and ParentProcessId

Reply:
So dropping the Testing.ParentProcessId